src/ck-core/cklocation.h

```diff
@@ -572,8 +572,10 @@ typedef std::unordered_map<CmiUInt8, CkMigratable*> ElemMap;
 	void initLB(CkGroupID lbmgrID, CkGroupID metalbID);
 
 public:
+#if CMK_LBDB_ON
```

Comment:
Why is there inconsistency between making a function a no-op (like above) and making a function undefined (like here)? I don't think this patch is the place to fix it, but I do think after the merge it would be good to cleanup CMK_LBDB_ON usage. It's wildly inconsistent from what I've seen and reaches all throughout the RTS.

Reply:
Agreed, there should be a big cleanup of all of the `CMK_LBDB_ON` usage. I'll look into it later.